sql/ddl_api.sql

```diff
@@ -29,7 +29,7 @@ CREATE OR REPLACE FUNCTION  create_hypertable(
     migrate_data            BOOLEAN = FALSE,
     chunk_target_size       TEXT = NULL,
     chunk_sizing_func       REGPROC = '_timescaledb_internal.calculate_chunk_interval'::regproc
-) RETURNS VOID AS '@MODULE_PATHNAME@', 'ts_hypertable_create' LANGUAGE C VOLATILE;
+) RETURNS TABLE(hypertable_id INT, schema_name NAME, table_name NAME) AS '@MODULE_PATHNAME@', 'ts_hypertable_create' LANGUAGE C VOLATILE;
```

Comment:
Some food for thought - should this just be the hypertable catalog table row?

Reply:
I had that idea too, but figured it would be quite jarring to output the whole row.